diff --git a/dist/mock.js b/dist/mock.js
index 3af239c4225fe3c25c2d4e7477b5c7762e0e8f13..c3e8372975db7ce720b0708d5c36397204fbdd31 100644
--- a/dist/mock.js
+++ b/dist/mock.js
@@ -777,6 +777,10 @@ return /******/ (function(modules) { // webpackBootstrap
 
 	Util.each('String Object Array RegExp Function'.split(' '), function(value) {
 	    Util['is' + value] = function(obj) {
+					if (value === 'Function') {
+							const type = Util.type(obj)
+							return type === 'function' || type === 'asyncfunction'
+					}
 	        return Util.type(obj) === value.toLowerCase()
 	    }
 	})
@@ -8375,7 +8379,7 @@ return /******/ (function(modules) { // webpackBootstrap
 	        if (this.custom.async) setTimeout(done, this.custom.timeout) // 异步
 	        else done() // 同步
 
-	        function done() {
+	        async function done() {
 	            that.readyState = MockXMLHttpRequest.HEADERS_RECEIVED
 	            that.dispatchEvent(new Event('readystatechange' /*, false, false, that*/ ))
 	            that.readyState = MockXMLHttpRequest.LOADING
@@ -8386,7 +8390,7 @@ return /******/ (function(modules) { // webpackBootstrap
 
 	            // fix #92 #93 by @qddegtya
 	            that.response = that.responseText = JSON.stringify(
-	                convert(that.custom.template, that.custom.options),
+	                await convert(that.custom.template, that.custom.options),
 	                null, 4
 	            )
 
@@ -8530,9 +8534,9 @@ return /******/ (function(modules) { // webpackBootstrap
 	}
 
 	// 数据模板 ＝> 响应数据
-	function convert(item, options) {
+	async function convert(item, options) {
 	    return Util.isFunction(item.template) ?
-	        item.template(options) : MockXMLHttpRequest.Mock.mock(item.template)
+	        await item.template(options) : MockXMLHttpRequest.Mock.mock(item.template)
 	}
 
 	module.exports = MockXMLHttpRequest